src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -233,7 +233,7 @@ function createEmscriptenModuleInstance(loadAssemblyUrls: string[], onReady: ()
 
   module.printErr = line => {
     console.error(`WASM: ${line}`);
-    let errorUi = document.querySelector("#error-ui") as HTMLElement;
+    let errorUi = document.querySelector('#error-ui') as HTMLElement;
```

Comment:
Could `MonoPlatform.ts` also call your `showErrorNotification` function from `BootErrors.ts` so it doesn't need its own copy of the logic?